specs/protocol/stage-1.md

```diff
@@ -42,3 +44,105 @@ not be able to perform any actions which have an impact on either the safety or
 system. An example of such a role is the `SystemConfig` owner (AKA Chain Operator), which can
 modify fees and other protocol parameters. Constraints MUST be placed on the allowable range of
 values for these parameters so that they do not result in a loss of liveness or safety.
+
+## Configuration of Safes
+
+This list outlines the various Safes, their extensions, and other configuration details necessary to
+achieve Stage 1.
+
+1. **The ProxyAdminOwner Safe:** This Safe acts as the Upgrade Controller. Accordingly, it is
+   authorized to call the following safety-critical functions:
+      - All `ProxyAdmin` `onlyOwner` functions.
+      - All `DisputeGameFactory` `onlyOwner` functions.
+      - All `DelayedWeth` `onlyOwners` functions.
+
+   This safe has a threshold of 2, and is owned by two other Safes:
+      1. The Security Council Safe.
+      2. The Foundation Upgrade Safe.
+
+   In general, the threshold and owner set of the safe must be configured such that an upgrade
+   or other safety-critical action can be performed without the cooperation of the Security Council.
```

Comment:
Is this correct? Since we do need the SC to perform upgrades (also "owner set" read confusingly to me, I thought "set" was a verb 😅)
```suggestion
   In general, the threshold and owners of the L1 ProxyAdmin Owner safe must be configured such that an upgrade
   or other safety-critical action can only be performed with the cooperation of the Security Council.
```

Reply:
Oops, it was meant to say "cannot"!

How about this change instead: 

```suggestion
   In general, the threshold and number of owners MUST be configured such that an upgrade
   or other safety-critical action can NOT be performed without the cooperation of the Security Council.
```

Reply:
That lgtm!